runtime/src/chpl-mem-replace-malloc.c

```diff
@@ -231,7 +231,7 @@ int posix_memalign(void **memptr, size_t alignment, size_t size)
     *memptr = NULL;
     ret = chpl_posix_memalign_check_valid(alignment);
     if( ret ) return ret;
-    *memptr = memalign(alignment, size);
+    *memptr = __libc_memalign(alignment, size);
```

Comment:
Looks like it would have called memalign in this file and that would have checked chpl_mem_inited() and then run __libc_memalign. But the tracking would have accounted for it twice, which is a bug I think.

Looks like a good change to me. Thanks!